android/src/main/java/com/plaid/gson/GsonTypeAdapters.kt

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (c) 2020 Plaid Technologies, Inc. <support@plaid.com>
+ */
+package com.plaid.gson
+
+import com.google.gson.JsonDeserializationContext
+import com.google.gson.JsonDeserializer
+import com.google.gson.JsonElement
+import com.google.gson.JsonObject
+import com.google.gson.JsonPrimitive
+import com.google.gson.JsonSerializationContext
+import com.google.gson.JsonSerializer
+import com.plaid.link.result.*
+import com.plaid.link.event.*
+import java.lang.reflect.Type
+
+class AccountAdapter : JsonSerializer<LinkAccount> {
+
+  override fun serialize(
```

Comment:
since these are all in the Android SDK already, I'm worried copy-pasting them will cause issues if we forget to update here. 

They serialize/deserialize public classes anyway so could we move them to `com.plaid.link.internal.adapters` which would be next to `BasePlaid` so they aren't obfuscated but they are in and `internal` package and then commented to not use (although since they're gson adapters there's no real harm if people do use them)

Reply:
Agreed, and we can tie the RN SDK to the correct Android SDK.

Fwiw, this is the one adapter that differs in RN vs Android (and making the Android ones visible will make that more apparent :))

Reply:
We could express that in the adapter class name?